regression/esbmc/github_591-2/main.c

```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+#include <string.h>
+int main()
+{
+	int z;
```

Comment:
Something is wrong with the code indentation.

Reply:
Not really, it's the Linux kernel coding style ;) Anyway, changed.